Algebraic-Diagrammatic-Construction/README.md

```diff
@@ -0,0 +1,19 @@
+Algebraic Diagrammatic Construction Theory
+==========================================
+Algebraic diagrammatic construction (ADC) theory is the excited-state counterpart
+to Moller-Plesset perturbation theory. ADC(n) allows the calculation of excitation
```

Comment:
*Møller*-Plesset 😄

Reply:
Fixed in https://github.com/psi4/psi4numpy/pull/105/commits/03e282e9dc5b4b1eee6758e340823c873c602bf6